driver-pulsar/src/main/java/io/nosqlbench/driver/pulsar/ops/PulsarOp.java

```diff
@@ -3,6 +3,12 @@
 /**
  * Base type of all Pulsar Operations including Producers and Consumers.
  */
-public interface PulsarOp extends Runnable {
+public interface PulsarOp {
```

Comment:
We want to keep this implementing Runnable directly, I think. I'll merge it for now to support your testing, but we should discuss the API strategy for this.